lttng-events.c

```diff
@@ -2464,6 +2466,8 @@ int _lttng_session_metadata_statedump(struct lttng_session *session)
 		uuid_c[8], uuid_c[9], uuid_c[10], uuid_c[11],
 		uuid_c[12], uuid_c[13], uuid_c[14], uuid_c[15]);
 
+	product_uuid = dmi_get_system_info(DMI_PRODUCT_UUID);
```

Comment:
missing NULL pointer check and handling.

Reply:
indeed. And question, the answer will apply also to UST since we can also add the product_uuid, but only if lttng-sessiond is run as root:

What would be preferable to do in case of null pointer: omit the product_uuid from the section or empty string?

Reply:
omit field from section.